includes/api/class-wc-admin-rest-reports-orders-controller.php

```diff
@@ -156,55 +156,61 @@ public function get_item_schema() {
 			'title'      => 'report_orders',
 			'type'       => 'object',
 			'properties' => array(
-				'order_id'       => array(
+				'order_id'         => array(
 					'description' => __( 'Order ID.', 'woocommerce-admin' ),
 					'type'        => 'integer',
 					'context'     => array( 'view', 'edit' ),
 					'readonly'    => true,
 				),
-				'order_number'   => array(
+				'order_number'     => array(
 					'description' => __( 'Order Number.', 'woocommerce-admin' ),
 					'type'        => 'string',
 					'context'     => array( 'view', 'edit' ),
 					'readonly'    => true,
 				),
-				'date_created'   => array(
+				'date_created'     => array(
 					'description' => __( 'Date the order was created.', 'woocommerce-admin' ),
 					'type'        => 'string',
 					'context'     => array( 'view', 'edit' ),
 					'readonly'    => true,
 				),
-				'status'         => array(
+				'date_created_gmt' => array(
```

Comment:
This new schema and `date_created` above are strings, but core has a `'type' => 'date-time',` for it's dates. Does this need adding?

Reply:
That was fixed in 40c867d531710b5e82f4174b19524be185e4ac18.